src/plots/polar/polar.js

```diff
@@ -556,36 +556,25 @@ proto.updateAngularAxis = function(fullLayout, polarLayout) {
 
     var out = Axes.makeLabelFns(ax, 0);
     var labelStandoff = out.labelStandoff;
-    var labelShift = out.labelShift;
-    var offset4fontsize = (angularLayout.ticks !== 'outside' ? 0.7 : 0.5);
-    var pad = (ax.linewidth || 1) / 2;
 
     var labelXFn = function(d) {
         var rad = t2g(d);
-
-        var offset4tx = signSin(rad) === 0 ?
-            0 :
-            Math.cos(rad) * (labelStandoff + pad + offset4fontsize * d.fontSize);
-        var offset4tick = signCos(rad) * (d.dx + labelStandoff + pad);
-
-        return offset4tx + offset4tick;
+        return Math.cos(rad) * labelStandoff;
     };
 
     var labelYFn = function(d) {
         var rad = t2g(d);
-
-        var offset4tx = d.dy + d.fontSize * MID_SHIFT - labelShift;
-        var offset4tick = -Math.sin(rad) * (labelStandoff + pad + offset4fontsize * d.fontSize);
-
-        return offset4tx + offset4tick;
+        var ff = Math.sin(rad) > 0 ? 0.2 : 1;
+        return -Math.sin(rad) * (labelStandoff + d.fontSize * ff) +
+            Math.abs(Math.cos(rad)) * (d.fontSize * MID_SHIFT);
```

Comment:
Do we support multi-line labels?

Reply:
Now we do -> https://github.com/plotly/plotly.js/pull/3538/commits/dab3a1f484166b7330d26ca2553780c49b8e2659

Thanks for reminding me!